src/tests/3_user.test.ts

```diff
@@ -63,6 +63,11 @@ describe("User service can", () => {
 		expect(res.status).to.equal(400);
 	});
 
+	it("can validate correct password", async () => {
+		const res = await userService.validatePassword(input);
+		expect(res).not.to.equal(undefined);
```

Comment:
if it is a correct match, I think we should expect the response to be true here. Let's add `expect(res).to.be.true;`

Reply:
it returns user object